src/components/Tabs/Tabs.st.css

```diff
@@ -1,10 +1,16 @@
 :import {
-    -st-from: "../../tpaStyles/styleableExt";
+    -st-from: "../../tpaStyles/styleableExt.js";
```

Comment:
is the `.js` needed (not according to the Stylable docs) https://stylable.io/docs/references/formatters